core/node/node_framework/src/implementations/layers/state_keeper/output_handler.rs

```diff
@@ -84,6 +89,38 @@ impl OutputHandlerLayer {
         self.protective_reads_persistence_enabled = protective_reads_persistence_enabled;
         self
     }
+
+    async fn validate_l2_legacy_shared_bridge_addr(
```

Comment:
In general, I don't think it's a good idea to either define business logic in `node_framework` (mentioned in https://github.com/matter-labs/zksync-era/pull/2949), or to run it during wiring (as opposed to a task).